CWE/events/soviet_union - Post Brezhnev stuff.txt

```diff
@@ -364,7 +364,7 @@ country_event = { # Replaced Administration
 		any_pop = {
 			ideology = { factor = 0.03 value = communist }
 		}
-		political_reform = open_unregulated
+		#political_reform = open_unregulated elitist_service
```

Comment:
I can't seem to edit this in the PR, could you help me add political_reform = elitist_service here?

Reply:
Sorry, the edit flag was turned off. I'll make sure to keep it on.